frame/clearing-house/src/lib.rs

```diff
@@ -917,6 +996,38 @@ pub mod pallet {
 			Self::deposit_event(Event::FundingUpdated { market: market_id.clone(), time: now });
 			Ok(())
 		}
+
+		#[transactional]
+		fn liquidate(
+			liquidator_id: &Self::AccountId,
+			user_id: &Self::AccountId,
+		) -> Result<(), DispatchError> {
+			let summary = Self::summarize_account_state(user_id)?;
+
+			let fees = Self::fully_liquidate_account(user_id, summary)?;
```

Comment:
Wouldn't be a case where liquidations would be partial?

Reply:
Yes, but not the scope of this PR